src/Login/Login.js

```diff
@@ -12,6 +14,11 @@ class Login extends Component {
   }
 
   render() {
+
+      if (this.props.loggedIn === true ) {
```

Comment:
Awesome we figured out this Redirect! This definitely makes more sense than wrapping the submit button in a Link like we originally had